apps/zotonic_core/src/support/z_logger_formatter.erl

```diff
@@ -497,7 +497,7 @@ pretty_stack(Stack, Config) ->
             [
                 "\n     ",
                 maybe_color(colored_mfa, Config),
-                io_lib:format("~tp:~tp/~p", [ M, F, Arity ]),
+                io_lib:format("~ts:~ts/~p", [ bin(M), bin(F), Arity ]),
```

Comment:
Maybe can be use `z_convert:to_binary/1` instead of custom `bin/1`?
```suggestion
                io_lib:format("~ts:~ts/~p", [ z_convert:to_binary(M), z_convert:to_binary(F), Arity ]),
```

Reply:
Maybe, for lists (string) I would like to handle them as unicode strings, where `z_convert` at the moment handles lists as utf8 `iodata`.

Reply:
The `z_convert:to_binary` is something I would like to change in a next big release, so that it assumes Unicode data if it is a list.